.travis.yml

```diff
@@ -63,6 +65,8 @@ script:
         case "$TRAVIS_JDK_VERSION" in
           "oraclejdk8")
             mvn verify -P dev -Dguava.version=$GUAVA_VERSION ;;
+          "openjdk6")
+            mvn verify -Dmaven.test.skip -Dguava.version=$GUAVA_VERSION ;;
           *)
```

Comment:
Yes. I thought it also was a good default, though.  Do you think it would be better to handle "oraclejdk7" as a separate case and fail in the default case?